tools/deployment-cli-tools/tests/resources/applications/myapp/deploy/values.yaml

```diff
@@ -6,6 +6,12 @@ harness:
       - legacy
     build:
       - cloudharness-flask
+    git:
+    - url: https://github.com/a/b.git
+      branch_tag: master
+    - url: https://github.com/c/d.git
+      branch_tag: v1.0.0
+      path: myrepo
```

Comment:
@filippomc question: the path is this the path that will be copied/mounted in the Docker build?

Reply:
No, I've finally opted for not making assumptions on where the files are getting copied inside the Dockerfile, there are too many variables to take into account as copying the whole repo is not usually a good idea. 

That's the path where the repo is cloned to inside `dependencies`. It's optional as we use the repo name by default.

Please let me know if it's clear from the docs https://github.com/MetaCell/cloud-harness/blob/feature/CH-37/docs/applications/dependencies.md#git-repository-dependencies

Reply:
question @filippomc : what is the usecase for having a path here? It can only cause errors if I have 2 different repos using the same path (when I make a mistake)

Reply:
Yes it's only about avoiding remote and potential name clashes. Could have left it out but was fast to implement :)